Containers/mastercontainer/Caddyfile

```diff
@@ -20,6 +20,13 @@ http://:80 {
   redir https://{host}{uri}
 }
 
+https://:8080 {
+        reverse_proxy localhost:8000
+        tls internal {
+                on_demand
```

Comment:
Will this still work also with ip-addresses now that https://github.com/nextcloud/all-in-one/blob/15da1035fb814c559b385219e1d992684e9075dc/Containers/mastercontainer/Caddyfile#L18-L20 is in place?

Reply:
not sure

Reply:
But I think internal tls ignores ask

Reply:
would need to be verified then...

Reply:
> But I think internal tls ignores ask

Yes they do